config/app/com/microsoft/azure/iotsolutions/uiconfig/services/Storage.java

```diff
@@ -233,9 +323,46 @@ public CompletionStage deleteDeviceGroupAsync(String id) throws BaseException {
 
         input.setDateCreated(Storage.DATE_FORMAT.print(DateTime.now().toDateTime(DateTimeZone.UTC)));
         final String value = toJson(input);
-        return client.createAsync(PackagesCollectionId, value).thenApplyAsync(p ->
-            Storage.createPackage(p)
-        );
+
+        CompletionStage<Package> result = client.createAsync(
+                PackagesCollectionId,
+                value)
+                .thenApplyAsync(p ->
+                    Storage.createPackage(p)
+                );
+
+        if (!(StringUtils.isBlank(input.getConfigType())) &&
+            input.getPackageType().equals(PackageType.deviceConfiguration))
+        {
```

Comment:
Nit: { goes on previous line

Reply:
Yes as discussed we will do this in different PR.

Reply:
I would recommend fixing easy things like this now

Reply:
ok